session/pingpong/factory.go

```diff
@@ -38,6 +38,12 @@ import (
 // DefaultAccountantFailureCount defines how many times we're allowed to fail to reach accountant in a row before announcing the failure.
 const DefaultAccountantFailureCount uint64 = 3
 
+// DefaultPaymentInfo represents the default payment info for the alpha release
+var DefaultPaymentInfo = dto.PaymentPerTime{
+	Price:    money.NewMoney(0.001, money.CurrencyMyst),
```

Comment:
My suggestion is to use decimal for money to avoid the unexpected in float to int conversions. Accountant, on the other hand, may use ints (which - per our discussion - I understood is used by blockchain).

Reply:
As talked - we'll just use uints here as we're aligning to what the blockchain prefers